packages/linter/src/migrations/update-10-4-0/update-eslint-configs-to-use-nx-presets.ts

```diff
@@ -393,7 +393,6 @@ const currentTypescriptConfig: Linter.Config = {
     'plugin:@typescript-eslint/eslint-recommended',
     'plugin:@typescript-eslint/recommended',
     'prettier',
-    'prettier/@typescript-eslint',
```

Comment:
This might be a bit confusing but I don't think this should be removed.

We want this config here to include `prettier/@typescript-eslint` because it will be deduped out of the user's config. If it is removed, I believe users will still have `prettier/@typescript-eslint` in their config which will throw an error.